modules/_labs/color-picker/react/spec/ColorPicker.spec.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+import {fireEvent, render} from '@testing-library/react';
+import {colors} from '@workday/canvas-kit-react-core';
+import {ColorPicker, ColorPickerProps} from '@workday/canvas-kit-labs-react-color-picker';
+
+describe('<ColorPicker /> component', () => {
+  const dataTestId = 'TESTY_TEST';
+
+  const renderColorPicker = (props?: Partial<ColorPickerProps>) => {
+    return render(<ColorPicker data-test-id={dataTestId} onChange={jest.fn()} {...props} />);
+  };
+
+  it('should include aria-label', () => {
+    const ariaLabel = 'Background Color';
+    const {container, unmount} = renderColorPicker({ariaLabel});
+
+    expect(container.querySelector(`[aria-label="${ariaLabel}"]`)).not.toBeNull();
+    unmount();
```

Comment:
React Testing Library gives us nice helper functions to find things

```suggestion
    const {getByLabelText} = renderColorPicker({ariaLabel});

    expect(getByLabelText(ariaLabel)).not.toBeNull();
```

But this won't give us a very helpful error messages if it fails. It will tell us it couldn't find any element that is labelled by `ariaLabel`. Another way is to do:

```tsx
    const {container} = renderColorPicker({ariaLabel});

    expect(container).toContain(`aria-label="${ariaLabel}"`)
```

which will show the HTML and state it could not find the string you provided (at least giving some context). This can be an extremely noisy error if the HTML is large.

If you have a bit more information of _which_ element you expect to have this label, you can encode it in the test. This will help offload some of your current context about this component into the test if this test ever fails.